CHANGELOG.md

```diff
@@ -1,5 +1,128 @@
+- <!-- 68 --> [DateRangePicker] Remove variant prop override for Textfield (#24433) @praveenkumar-kalidass
+- <!-- 77 --> [lab] Reflect draft pattern of picker value in implementation (#24367) @eps1lon
+
+### `@material-ui/styled-engine@`
+
+- <!-- 67 --> [styled-engine] Rename StylesProvider to StyledEngineProvider (#24429) @mnajdova
+
+### `@material-ui/system@`
+
+- <!-- 36 --> [system] Fix handling of null-ish values (#24530) @oliviertassinari
+
+### `@material-ui/unstyled@`
+
+- <!-- 72 --> [unstyled] Convert composeClasses to TypeScript (#24396) @eps1lon
+
+### `@material-ui/utils@`
+
+- <!-- 20 --> [utils] `useEventCallback` `args` defaults to `unknown[]` (#24564) @eps1lon
+- <!-- 69 --> [utils] Fix requirePropFactory to merge validators (#24423) @mnajdova
+
+### core
+
+- <!-- 29 --> [core] Improve pseudo classes overrides error (#24535) @mnajdova
+- <!-- 45 --> [core] Fix styleProps to always contain all props (#24505) @mnajdova
+- <!-- 56 --> [core] Do not forward classes prop by default in experimentalStyled (#24451) @mnajdova
+- <!-- 57 --> [core] Pass styleProps on all slots in the styled() components (#24454) @mnajdova
+- <!-- 58 --> [core] Batch small changes (#24445) @oliviertassinari
+- <!-- 79 --> [core] Normalize generating declaration files (#24411) @eps1lon
+
+### docs
+
+- <!-- 02 --> [docs] Add missing newline in component JSDOC (#24610) @eps1lon
+- <!-- 10 --> [docs] Add API of picker components (#24497) @eps1lon
+- <!-- 17 --> [examples] Add `locale` prop to the Nextjs Link component (#24596) @CyanoFresh
+- <!-- 28 --> [docs] List required props first in /api/* (#24526) @eps1lon
+- <!-- 35 --> [docs] Mention the system props when available in the API pages (#24503) @mnajdova
+- <!-- 37 --> [docs] Improve system properties page (#24524) @mnajdova
+- <!-- 38 --> [docs] Fix malformed component API description (#24504) @eps1lon
+- <!-- 39 --> [docs] Fix ToC "Component name" fragment link on /api/* (#24517) @eps1lon
+- <!-- 40 --> [docs] Fix ToC on /api pages linking to api-docs (#24515) @eps1lon
+- <!-- 43 --> [docs] Add comment explaining specificity bump on Select (#24509) @KarimMokhtar
+- <!-- 52 --> [docs] Compute spreadable from tests (#24490) @eps1lon
+- <!-- 53 --> [docs] Fix label bug in stepper vertical demo (#24491) @artola
+- <!-- 60 --> [docs] Update Divjoy URL (#24447) @mbrookes
+- <!-- 63 --> [docs] Improve packages description (#24330) @oliviertassinari
+- <!-- 66 --> [docs] Fix content-layout-shift (#24418) @oliviertassinari
+- <!-- 71 --> [docs] Document default values of external props (#24417) @eps1lon
+- <!-- 78 --> [docs] Update in-house ads (#24410) @mbrookes
+
+### test
+
+- <!-- 21 --> [test] Simplify DatePicker tests (#24545) @eps1lon
+- <!-- 46 --> [test] Fix AvatarGroup failing test (#24512) @mnajdova
+- <!-- 55 --> [test] Split tests in describeConformanceV5 to isolate them (#24479) @mnajdova
+- <!-- 51 --> [pickers] Sort tests (#24481) @eps1lon
```

Comment:
Pickers is part of the lab

Reply:
It's just tests and I don't think we should change these items. They're just a copy of the commit message.

Reply:
I agree that we should aim to not change the commit message. We have made `pickers` the main label of this change by putting it into the bracket. I don't understand how it can be under the `test` section.